src/scala/com/github/johnynek/bazel_deps/Writer.scala

```diff
@@ -70,14 +70,28 @@ object Writer {
         val actual = Label.externalJar(l, coord.unversioned)
         def kv(key: String, value: String): String =
           s""""$key": "$value""""
-        List(s"""$comment    callback({${kv("artifact", coord.asString)}""",
+        List(s"""$comment    declare_maven({${kv("artifact", coord.asString)}""",
              s"""${kv("lang", l.asString)}$shaStr$serverStr""",
              s"""${kv("name", coord.unversioned.toBazelRepoName)}""",
              s"""${kv("actual", actual.asStringFrom(Path(Nil)))}""",
              s"""${kv("bind", coord.unversioned.toBindingName)}})""").mkString(", ")
       }
       .mkString("\n")
-    s"""def maven_dependencies(callback):\n$lines\n"""
+    s"""def declare_maven(hash):
+        |    native.maven_jar(
+        |        name = hash["name"],
+        |        artifact = hash["artifact"],
+        |        sha1 = hash["sha1"],
+        |        repository = hash["repository"]
+        |    )
+        |    native.bind(
+        |        name = hash["bind"],
+        |        actual = hash["actual"]
+        |    )
+        |
+        |def maven_dependencies():
```

Comment:
this isn't quite what I meant, sorry.

Can you change this to:
```python
def maven_dependencies(callback = declare_maven):
```
so then people can override or not?